client/components/credit-card/stored-card.jsx

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+class StoredCard extends React.Component {
+	static propTypes = {
+		card: PropTypes.string.isRequired,
+		card_type: PropTypes.string.isRequired,
+		name: PropTypes.string.isRequired,
+		expiry: PropTypes.string.isRequired,
+	};
+
+	render() {
+		const { card, card_type, name, expiry } = this.props;
+		const expirationDate = this.props.moment( expiry ).format( 'MM/YY' );
```

Comment:
while already discussed and established that this is a standard format for credit card expiration months it's not obvious to the future reader why we aren't using a locale-based format string.

can we add an explanatory comment here to prevent future changes that would break the behavior?

```js
// the use of `MM/YY` is standard across credit-card forms and should not be localized
const expirationDate = this.props…
```

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24485/commits/bdcbd88445f8a34648b8cc2c0b7c0c0d233f0b6d